.gitignore

```diff
@@ -1,20 +1,7 @@
-# The directory Mix will write compiled artifacts to.
-/_build/
-
-# If you run "mix test --cover", coverage assets end up here.
-/cover/
-
-# The directory Mix downloads your dependencies sources to.
-/deps/
-
-# Where 3rd-party dependencies like ExDoc output generated docs.
-/doc/
-
-# Ignore .fetch files in case you like to edit your project deps locally.
-/.fetch
-
-# If the VM crashes, it generates a dump, let's ignore it too.
-erl_crash.dump
-
-# Also ignore archive artifacts (built via "mix archive.build").
-*.ez
+**/*
+!*.*
+!bash
+!elixir
+!python
+!ruby
```

Comment:
I wonder if we should eventually move these into a `current` folder so we don't have to do this for every iteration. I'm not planning on supporting more languages yet though so this is good enough.

Reply:
`go` version is coming soon